CHANGELOG.rst

```diff
@@ -2,6 +2,26 @@
 Changelog
 =========
 
+2.6.3 (2024-04-10)
+==================
+
+Bugfix release following feeback on 2.6.2
```

Comment:
Missing 'd' in feedback (in case you have something else to add/change to this PR)